lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -276,6 +352,24 @@ LogicalResult CreateSiFiveMetadataPass::emitRetimeModulesMetadata() {
   if (filename.empty())
     return success();
 
+  auto unknownLoc = mlir::UnknownLoc::get(context);
```

Comment:
I think it's reasonable to use unknown loc for schema, but can we use concrete locations for memory fields?

Reply:
Makes sense, used the op locations.